gateway/ln-gateway/proto/gatewaylnrpc.proto

```diff
@@ -110,7 +98,7 @@ message SubscribeInterceptHtlcsResponse {
   uint64 htlc_id = 13;
 }
 
-message CompleteHtlcsRequest {
+message CompleteHtlcResponse {
```

Comment:
maybe we should rename to `InterceptHtlcResponse` given it now includes `Forward` outcome

Reply:
I like that, will change